internal/server/v1/page/golden/place_page/asm.Economics.json

```diff
@@ -996,38 +1002,21 @@
         },
         "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity": {
           "val": {
-            "1990": 184590918560.761,
-            "1991": 192266777369.145,
-            "1992": 203019674453.407,
-            "1993": 198250043195.704,
-            "1994": 202332384993.974,
-            "1995": 211019103740.608,
-            "1996": 225608775588.983,
-            "1997": 236936583685.765,
-            "1998": 242830643991.792,
-            "1999": 255276962389.21,
-            "2000": 254877365300.465,
-            "2001": 286931665309.501,
-            "2002": 332767928184.343,
-            "2003": 360446875994.608,
-            "2004": 404242860236.613,
-            "2005": 441866754015.892,
-            "2006": 515363004319.01,
-            "2007": 549527714436.745,
-            "2008": 596601492158.76,
-            "2009": 645018537807.676,
-            "2010": 705514512939.028,
-            "2011": 756939368351.467,
-            "2012": 793230176344.865,
-            "2013": 853026950842.773,
-            "2014": 939073115447.917,
-            "2015": 957051778143.622,
-            "2016": 952515802712.001,
-            "2017": 960398587580.009,
-            "2018": 986320858012.442,
-            "2019": 1040597740019.16,
-            "2020": 1031251405329.87,
-            "2021": 1114605112222.08
+            "2008": 596822437106.174,
```

Comment:
CL#610677906 World Development Indicator refresh : Earlier years missing for some countries like NGA from source.